stylesheet.css

```diff
@@ -490,120 +490,128 @@ table {
 
 /*new menu*/
 
-.nav>li>a {
-    position: relative;
-    display: block;
-    padding: 10px 8px;
```

Comment:
wysokość navbaru jest inna niż całkowita wysokość przycisków menu razem z paddingami. źle przez to wygląda hover